nodepool/nodepool.yaml

```diff
@@ -62,6 +62,13 @@ labels:
     min-ready: 1
     providers:
       - name: devstack-rocket-man-asr
+  - name: centos-7-asr-prll
+    image: centos-7
```

Comment:
if you update the image name based on my comment below this will need updating too

Reply:
sure